### ISSUE TITLE
Cant move to other screen

### ISSUE THREAD

**Issue by Xapphire13** [singuerinc/overlay#145]
It seems like the Overlay app is constrained to only the primary screen

**Comment by singuerinc** [comment]
@Xapphire13 you are right, Overlay creates a "transparent window" in the current screen that serves as limit/space where you can drag the tools. I can't think right now in a way to achieve this.

Since I'm using [Slate](https://github.com/jigish/slate) (a window manager) to move windows from one screen to another I did not realized that you can't do it with the mouse.

I'll take a look, thanks for reporting it.
